### PR TITLE
Fix rubocop warnings

### PR DESCRIPTION
**Pull request by andy-faunadb**

### CODE REVIEW

File: lib/fauna/connection.rb [hunk 1]
```diff
@@ -35,7 +35,7 @@ class Connection
     #            +:connection_timeout+:: \Connection open timeout in seconds.
     #            +:adapter+:: Faraday adapter to use. Either can be a symbol for the adapter, or an array of arguments.
     #            +:secret+:: Credentials to use when sending requests. User and pass must be separated by a colon.
-    def initialize(params = {}) # rubocop:disable Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity, Metrics/MethodLength
+    def initialize(params = {}) # rubocop:disable Metrics/CyclomaticComplexity, Metrics/MethodLength, Metrics/PerceivedComplexity
```
Comment (ashfire908) [inline lib/fauna/connection.rb]: What's this for?

File: test/query_test.rb [hunk 2]
```diff
@@ -51,40 +51,36 @@ def test_quote
...
       lambda: 'auto0',
       expr: { add: [{ var: 'auto0' }, { var: 'auto0' }] }
-    }
 
     lambda = Fauna::Query.lambda do |a|
       Fauna::Query.lambda do |b|
         Fauna::Query.lambda { |c| [a, b, c] }
       end
     end
-    assert_equal lambda, {
```
Comment (ashfire908) [inline test/query_test.rb]: Is that removal of `{` and adding of `,` to last element both warnings if not done?

Comment (andy-faunadb) [inline test/query_test.rb]: Rubocop warns if the last parameter is a hash and you put in unnecessary curly braces.
For hashes with curly braces, it wants you to put a comma on the last line.

**Comment by ashfire908** [comment]
LGTM

**Pull request merged**